src/Styleguide/StyledSystemTypes.ts

```diff
@@ -0,0 +1,26 @@
+export interface MarginProperties {
```

Comment:
I don't think this is the best place for this file... It should probably go in typings? I also don't know exactly what to name it.